MODULE.bazel

```
@@ -0,0 +1,25 @@
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+module(
+    name = "opentelemetry-cpp",
+    version = "1.14.2",
```

```suggestion
    version = "1.15.0",
```

I assume this should match the current version in the opentelemetry-cpp repository.

---

actually this number just needs to be replaced when it's uploaded to the BCR with a patch, so I actually just zero'd this out